packages/Popover/src/Popover.js

```diff
@@ -26,6 +27,10 @@ const focusableElementSelector =
 // TODO: To handle cases where there are multiple scrolling containers, we need to implement
 //       getScrollContainer as oneOfType([func, arrayOf(func)])
 
+function isDescendantOfActiveElement() {
+  return isDescendant(document.activeElement, "data-pka-anchor", "popover.content");
```

Comment:
Both this case and Overlay don't uniquely describe the instance of a component (e.g. there can be many popovers on the page). I can't think of a specific scenario where this would be an issue but thought I'd note it here.

Reply:
Yes my thinking was that we only should have one popover open and taking focus at any time, and opening one should always close another.. At least that is how it Should be :)